connector/signaltometricsconnector/factory.go

```diff
@@ -32,26 +38,83 @@ func createDefaultConfig() component.Config {
 func createTracesToMetrics(
 	_ context.Context,
 	set connector.Settings,
-	_ component.Config,
+	cfg component.Config,
 	nextConsumer consumer.Metrics,
 ) (connector.Traces, error) {
-	return newSignalToMetrics(set, nextConsumer), nil
+	c := cfg.(*config.Config)
+	parser, err := ottlspan.NewParser(customottl.SpanFuncs(), set.TelemetrySettings)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create OTTL statement parser for datapoints: %w", err)
```

Comment:
Is `"datapoints"` accurate here or it should be `"spans"`?
```suggestion
		return nil, fmt.Errorf("failed to create OTTL statement parser for spans: %w", err)
```